ubuntu-core-20-intel-iot-beta.json

```
@@ -0,0 +1,38 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "revision": "1",
+    "model": "ubuntu-core-20-intel-iot-beta",
```

```suggestion
    "model": "ubuntu-core-20-intel-iot-beta",
```

If this has been already submitted for signing then as xnox mentioned, let's just leave it - but otherwise I think it's best to just get rid of `revision` as 0 is implied. I don't think we need to have a bump in the revision?